scripts/deploy.ts

```diff
@@ -84,6 +84,16 @@ class DeploymentExecutor {
       event.args._approved
     );
 
+    tx = await this.vouchers.setContractUri(process.env.VOUCHERS_CONTRACT_URI);
```

Comment:
After merging #262, add `this.txOptions as final argument to this function, like so:

```javascript
tx = await this.vouchers.setContractUri(process.env.VOUCHERS_CONTRACT_URI, this.txOptions);
```

Reply:
The above PR is now merged, and tested on rinkeby, i guess we are good to go on his side of things now.

Reply:
I have changed the `scripts/deploy.ts` where `this.txOptions` isn't available.
`this.txOptions` is for the `deploy.ts` file where we do not deploy `Vouchers` and `VoucherSets`.

Reply:
`deploy.ts` in root was created in https://github.com/bosonprotocol/contracts/pull/262 but that file is not something that `npx hardhat deploy` (or `npm run contracts:migrate`) actually use. That `deploy.ts` is also obsolete (has still `erc1155erc721` contract and does not set contract addresse via constructors).

Actually this `scripts/deploy.ts` should have `this.txOptions` since this is the script used in deployment.

Reply:
Done in https://github.com/bosonprotocol/contracts/pull/261/commits/b93e54d0836fb56d447bc93d3a5565fe58f0e22d

Reply:
@zajck can you resolve this conversation, given timezones with cliff and given you commented on this ?

Reply:
deploy.ts should not be in the root directory

Reply:
@bidyut-arianelabs ^^

Reply:
Removed in https://github.com/bosonprotocol/contracts/pull/261/commits/facd919b6636b44e5c29dc1b80708d3b3fe77a99